be/test/vec/function/function_compressed_materialization_test.cpp

```diff
@@ -0,0 +1,207 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include <gtest/gtest.h>
```

Comment:
Maybe you could try to remove this head file safely, so that the git robot will not be annoying anymore. ^_^

Reply:
it the problem of bot, not of code ...

Reply:
Yes, I know. What I mean is we can avoid this by removing the line, if we don't have to resolve this every time before fixing the bot. Indeed, it is not a big deal.

Reply:
In principal, each cpp file should include whatever it need.

Reply:
What if we remove the line, the test would be failed or others? I could run some BE UTs without including the gtest.

Reply:
Do not take the problem to a larger scale. Just focus on this pr.

This cpp file uses gtest, so it adds it to the header, that is all.

Reply:
Please forgive my possible misleading at  the beginning. 
What I try to demonstrate is that you can try replacing with the test util head file to avoid the bot warning, not just remove the head file only.

Reply:
> Please forgive my possible misleading at the beginning. What I try to demonstrate is that you can try replacing with the test util head file to avoid the bot warning, not just remove the head file only.

I get you bro. But I don't want to change the code just because of the "bug" of the bot. For my opinion, the current git-action bot is terrible. Just leave them alone.

Reply:
> I find most existing BEUT for function without including the gtest.h file. To be more specific, those file use `function_test_util.h`, which has been including not only some gtest head files but also some useful functions like `check_function`, to test. So, if we use that, would it be a better choice?

What if `function_test_util.h` is refactored and no longer include gtest?